src/bindings/js/node/include/helper.hpp

```diff
@@ -18,6 +18,8 @@ const std::vector<std::string>& get_supported_types();
 
 typedef std::variant<napi_valuetype, napi_typedarray_type, js_type> napi_types;
 
+const std::unordered_map<napi_typedarray_type, ov::element::Type>& get_ov_type();
```

Comment:
```suggestion
const ov::element::Type get_ov_type(const napi_typedarray_type& napi_type);
```
To return the `OV` type corresponding to `napi_type`

Current solution returns map not ov type.